Renamed example to examples.

JulianFeinauer opened the pull request:
Added Python Quickstart Example.
Added Python Example to the getting-started page.

Signed-off-by: Julian Feinauer <j.feinauer@pragmaticminds.de>

sophokles73 commented on examples/quickstart-python/README.md:
```diff
@@ -0,0 +1,53 @@
+
+    SPDX-License-Identifier: EPL-2.0
+ -->
+# Eclispe Hono Quickstart tutorial
```
Eclipse

JulianFeinauer replied:
Fixed.

sophokles73 commented on examples/quickstart-python/pom.xml:
```diff
@@ -0,0 +1,33 @@
+    <description>This is a python example. See the Readme for more information.</description>
+    <url>https://www.eclipse.org/hono</url>
+
+</project>
```
missing end-of-line

JulianFeinauer replied:
Fixed.

sophokles73 commented on examples/quickstart-python/README.md:
```diff
@@ -0,0 +1,53 @@
+
+    SPDX-License-Identifier: EPL-2.0
+ -->
+# Eclispe Hono Quickstart tutorial
```
Eclipse

JulianFeinauer replied:
Fixed.

sophokles73 commented on examples/quickstart-python/src/main/python/quickstart.py:
```diff
@@ -0,0 +1,131 @@
+
+
+# Prepare the container
+uri = f'amqp://{amqpNetworkIp}:15672/telemetry'
```
I do not think that you need to append `/telemetry` here ..

JulianFeinauer replied:
Indeed, it works without and I changed it accordingly.

sophokles73 commented on site/homepage/content/getting-started/index.md:
```diff
@@ -35,6 +35,25 @@ The Hono command line client is used in this guide for simulating an application
 
+#### Very quick Quickstart
+
+In the following all necessary commands and operations will be explained on how to register a tenant and a device via the HTTP Api.
```
... the HTTP API.

JulianFeinauer replied:
Changed.

sophokles73 commented on site/homepage/content/getting-started/index.md:
```diff
@@ -35,6 +35,25 @@ The Hono command line client is used in this guide for simulating an application
+#### Very quick Quickstart
+
+In the following all necessary commands and operations will be explained on how to register a tenant and a device via the HTTP Api.
+Then, it will be shown how messages can be sent via the HTTP and MQTT Endpoint.
```
... via the HTTP and MQTT protocol adapters.

JulianFeinauer replied:
Changed.

JulianFeinauer commented:
@sophokles73 sorry for the typos. I changed all points accordingly and squashed it to one commit.

JulianFeinauer commented:
@sophokles73 thanks for the hint! I replaced all mentions and also fixed some (older) outdated links, so I hope the whole site structure is now fine with new repo example layout.

JulianFeinauer commented:
Just had a look into Travis and it seems that its not related to my changes but rather an issue with docker / redhats repo?

sophokles73 commented:
@dejanb @ctron anything to add?

sophokles73 commented:
@JulianFeinauer can you rebase and resolve the conflicts?
@ctron do you believe we need Python related works-with CQs for this?

sophokles73 commented:
@JulianFeinauer sorry for the delay with this CQ. However, we would like to merge it now so it would be great if you could rebase and remove the conflicts one final time. Thanks a lot for your patience.

ctron commented:
> @ctron do you believe we need Python related works-with CQs for this?

No, I don't think so.
